src/components/App.js

```diff
@@ -25,29 +25,29 @@ class App extends Component {
         Phone: "+44 7894561230"
       }
     }
+  });
+  const ChangePage = CurrentPage => {
+    setData({ CurrentPage });
   };
-  ChangePage = CurrentPage => {
-    this.setState({ CurrentPage });
+  const SetSection = (Section, Content) => {
```

Comment:
The function has been changed but have you updated the calls as well? I couldn't find them. This might potentially break the whole application.

Reply:
I have checked it within other files also, modification of this function doesn't change anything.

Reply:
Great, GitHub messed it up actually! 😊 It looks good.